onnxruntime/core/session/provider_stubs.cc

```diff
@@ -202,4 +202,11 @@ ORT_API(void, OrtApis::ReleaseTensorRTProviderOptions, _Frees_ptr_opt_ OrtTensor
   ORT_UNUSED_PARAMETER(ptr);
 }
 
+ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider_MIGraphX1,
```

Comment:
should the name here be updated?

Reply:
@weixingzhang Sorry for missing this one. Just fixed it. Thanks